tests/integration/helpers.py

```diff
@@ -18,3 +27,62 @@ async def get_leader_unit(model, application_name: str) -> Unit:
         if unit.application == application_name and await unit.is_leader_from_status():
             return unit
     raise RuntimeError(f"Leader unit for `{application_name}` not found.")
+
+
+async def get_unit_status_messages(
+    ops_test: OpsTest, app_name: str = APP_NAME
+) -> List[tuple[str, str]]:
+    """Get the status messages from all the units of the given application."""
```

Comment:
Can you specify what is in each tuple? I know reading the code tells it but I feel it'd be useful to know that it's <unit name>, <unit status> with an example maybe to know what we expect

Reply:
https://github.com/canonical/vault-k8s-operator/pull/429/commits/e877ef39c6ba4eb4b2d8d5f740f2e2aafb6a92b2